.github/workflows/template-notebook-test/action.yaml

```diff
@@ -0,0 +1,53 @@
+name: Notebook test template
+description: A composite action to setup and run example notebooks using Papermill
+
+inputs:
+  python-version:
+    required: false
+    description: Python version
+    default: "3.9"
+  papermill-args-yaml:
+    description: 'Additional arguments to pass to Papermill in yaml format'
+    required: false
+    default: ""
+  notebook-input:
+    description: 'List of paths to the input notebooks, separated by commas'
+    required: true
+
+runs:
+  using: composite
+  steps:
+
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ inputs.python-version }}
+
+    - name: Install dependencies
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip
+        pip install papermill kubeflow-katib jupyter ipykernel
+        python -m ipykernel install --user --name python3 --display-name "Python 3"
+
+    - name: Setup Minikube Cluster
+      shell: bash
+      run: ./test/e2e/v1beta1/scripts/gh-actions/setup-minikube.sh true true "" "" "cmaes"
```

Comment:
Could we reuse `template-setup-e2e-test`? I guess it will be better if we make full use of the existing template :)

Wait for you thoughts👀 @yehudit1987 @kubeflow/wg-automl-leads

Reply:
SGTM:)